.pylintrc

```diff
@@ -151,6 +151,8 @@ disable=print-statement,
         too-many-lines,
         too-many-instance-attributes,
         too-few-public-methods,
+        unsubscriptable-object,
```

Comment:
Where were we getting these errors?

Reply:
https://circleci.com/gh/plotly/dash-bio/5997?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
@Marc-Andre-Rivet Oh, this is why... https://circleci.com/gh/plotly/dash-bio/5997?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Added back the rule as https://circleci.com/gh/plotly/dash-bio/6024 demonstrates that this passed on master because it wasn't running the expect version of pylint (2.3.0 instead of 1.9.4) which has different defaults.